### PR TITLE
Switch COS Lite How-To to use the `k8s-operator` charm

### PR DESCRIPTION
**Pull request by mateoflorido**
## Overview
Update the `Integrating with COS Lite` How-To guide to use the `k8s-operator` charm as the deployment substrate instead of the MicroK8s snap.

### CODE REVIEW

**Review by bschimke95 — APPROVED** [review]
LGTM, leave the final approval to @nhennigan

File: docs/src/charm/howto/cos-lite.md [hunk 1]
```diff
@@ -77,9 +72,9 @@ juju offer prometheus:receive-remote-write
 Use `juju status --relations` to verify that both `grafana` and `prometheus`
 offerings are listed.
 
-At this point, you’ve established a MicroK8s model on Ubuntu and incorporated
-it into Juju as a Kubernetes cloud. You then used this cloud as a substrate for
-the COS Lite deployment. You therefore have 2 models on the same controller.
+At this stage, you’ve set up a Kubernetes cluster, registered it as a Juju
+cloud, and deployed COS Lite on it. This creates tow models on the same
```
Comment (louiseschmidtgen) [inline docs/src/charm/howto/cos-lite.md:76]:
```suggestion
cloud, and deployed COS Lite on it. This creates two models on the same
```

**Review by louiseschmidtgen — COMMENTED** [review]
Nice work Mateo! Couple of small nits.

I can't highlight this in the review but can you fix line 12 as well please:
This document assumes **that** you have a controller with an installation of Canonical
Kubernetes.

File: docs/src/charm/howto/cos-lite.md [hunk 2]
```diff
@@ -19,54 +19,49 @@ If you are unfamiliar with Juju models, the documentation can be found
...
 ```
 
-Deploy MicroK8s on Ubuntu by accessing the unit you created at the last step
-with `juju ssh microk8s/0` and following the 
-[Install MicroK8s][how-to-install-MicroK8s] guide for configuration.
-
-```{note} Make sure to enable the hostpath-storage and MetalLB addons for 
-MicroK8s.
+```{note} Make sure to enable the local-storage and load-balancer features for
+the `k8s` charm. These are essential for the COS Lite to function correctly.
```
[the remark this answers is not in the document]
Comment (mateoflorido) [reply docs/src/charm/howto/cos-lite.md:38]: We have a proper mechanism to enable these features through charm config. Let me update it!

File: docs/src/charm/howto/cos-lite.md [hunk 3]
```diff
@@ -9,64 +9,60 @@ first-class integration between {{product}} and COS Lite (Canonical
...
 ```
-KUBECONFIG=microk8s-config.yaml juju add-k8s microk8s-cloud
+KUBECONFIG=./kubeconfig juju add-k8s k8s-cloud
 ```
 
-## Deploying COS Lite on the MicroK8s cloud
+## Deploying COS Lite on the K8s cloud
 
-On the MicroK8s cloud, create a new model and deploy the `cos-lite` bundle:
+On the COS cloud, create a new model and deploy the `cos-lite` bundle:
```
Comment (amc94) [inline docs/src/charm/howto/cos-lite.md:58]: on the cos cloud or the k8s cloud?

File: docs/src/charm/howto/cos-lite.md [hunk 4]
```diff
@@ -9,64 +9,60 @@ first-class integration between {{product}} and COS Lite (Canonical
 Observability Stack). This guide will help you integrate a COS Lite
 deployment with a {{product}} deployment.
 
-This document assumes you have a controller with an installation of Canonical
-Kubernetes. If you have not yet installed {{product}}, please see
+This document assumes that you have a controller with an installation of
+Canonical Kubernetes. If you have not yet installed {{product}}, please see
```
Comment (nhennigan) [inline docs/src/charm/howto/cos-lite.md:13]: {{product}}

**Review by nhennigan — APPROVED** [review]
Small nit but otherwise LGTM. Once you answer amc94 question I think its good to merge.

File: docs/src/charm/howto/cos-lite.md (hunk 3 above)
Comment (mateoflorido) [reply docs/src/charm/howto/cos-lite.md:58]: `k8s` cloud. Thanks!

**Pull request merged**